src/components/AQIGauge/AQIDial.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {Text, Box, Tag, Link, Center} from '@chakra-ui/react';
+import GaugeSVG from './GaugeSVG';
+/**
+ * Interface for the props of the dial
+ * - currentReading is the AQI value that the dial should display
+ */
+interface DialProps {
+  currentReading: string;
+}
+
+/**
+ * AQI Dial Label Component
+ * This component displays the label for the AQI displayed by the dial.
+ * Ex if the AQI is less than 50, it will show the label "Good"
+ */
+const AQILabel: ({currentReading}: DialProps) => JSX.Element = ({
+  currentReading,
+}: DialProps) => {
+  // Convert the AQI from a string to a number
+  const aqi: number = +currentReading;
+  // AQI boundary values
+  const good = 50; // Air quality is good (0-50)
+  const moderate = 100; // Air quality is acceptable (51-100)
+  const sensitiveGroups = 150; // Health risk for sensitive groups (101-150)
+  const unhealthy = 200; // Health risk for all individuals (151-200)
+  const veryUnhealthy = 300; // Very unhealthy for all individuals (201-300)
+
+  if (aqi <= good) {
+    return (
+      <Tag fontSize={16} px={5} py={1} bg="#08E400" textColor="white">
```

Comment:
This is mostly the same code repeated, perhaps have variables named `backgroundColor`, `textColor`, `label` and `px` that are set in each if statement, and then return the
```
<Tag fontSize={16} px={px} py={1} bg={backgroundColor} textColor={textColor}>
        {label}
</Tag>
```